src/components/Calculator.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import Result from './Result';
+import Buttons from './Buttons';
+
```

Comment:
- [ ] Props need to get passed to child components of the `Calculate.js` component, this should happen inside the calculator component, please make the necessary changes in order to comply with the project requirements.